src/snipe/cli/cli_ops.py

```
@@ -868,3 +868,206 @@
         logger.error(f"Failed to export common hashes signature: {e}")
         click.echo(f"Error: Failed to export common hashes signature: {e}", err=True)
         sys.exit(1)
+
+import csv
```

Ruff

Module level import not at top of file

(E402)